lib/Wikia/src/Service/User/Permissions/PermissionsService.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace Wikia\Service\User\Permissions;
+
+interface PermissionsService {
+
+	/**
+	 * Gets the object containing definitions of groups and permissions in the system
+	 * @return PermissionsConfiguration
+	 */
+	public function getConfiguration();
+
+	/**
+	 * @param \User $user
+	 * @return string[] Get list of global and local groups to which the given user has been explicitly assigned to.
+	 */
+	public function getExplicitGroups(\User $user );
```

Comment:
Space mismatch inside parens (no space at the beginning vs one at the end), autoformat this file?